CommandLine/emake/OptionsParser.cpp

```diff
@@ -380,8 +382,8 @@ int OptionsParser::printInfo(const std::string &api)
 
 void OptionsParser::printHelp()
 {
-  outputStream << "Enigma Command Line Compiler" << std::endl
-            << _desc << std::endl;
+  outputStream << "Enigma Command Line Compiler" << std::endl << _desc << std::endl;
+  std::cout    << "Enigma Command Line Compiler" << std::endl << _desc << std::endl;
```

Comment:
We shouldn't need the `cout` copy; `outputStream` is suppressed because fundies deliberately grounded it. He should have done that after help printing finished...